Packs/Celonis/Integrations/CelonisEventCollector/CelonisEventCollector_description.md

```diff
@@ -0,0 +1,20 @@
+## Celonis Help
+
+#### How to create an OAuth client and generate client ID and Client Secret:
+[Link to Celonis Audit Logs Documentation.](https://developer.celonis.com/celonis-apis/audit-log-api/#creating-an-application-and-granting-it-api-permissions)
```

Comment:
Add this below the explanation. 
With the text - For more information visit Celonis Audit Logs Documentation